certified-connectors/World Academia/apiProperties.json

```diff
@@ -0,0 +1,11 @@
+{
+  "properties": {
+    "connectionParameters": {},
+    "iconBrandColor": "#C1FF72",
+    "capabilities": [
+      "actions"
+    ],
+    "publisher": "Kelcho Tech",
```

Comment:
Hi @kelcho-spense 
Please make necessary changes as per below. 
Publisher should be - Name of company or organization
stackOwner should be - Name of the owner of the service or product.
And please refer the template link here. https://github.com/microsoft/PowerPlatformConnectors/blob/dev/templates/certified-connectors/apiProperties.json

Reply:
Hello @vmanoharas Please review my changes, I have fixed the stackOwner